java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -288,6 +289,51 @@ public void getdel(BaseClient client) {
         assertInstanceOf(RequestException.class, executionException.getCause());
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void getex(BaseClient client) {
+
+        assumeTrue(REDIS_VERSION.isGreaterThanOrEqualTo("6.2.0"), "This feature added in redis 6.2.0");
+
+        String key1 = "{key}" + UUID.randomUUID();
+        String value1 = String.valueOf(UUID.randomUUID());
+        String key2 = "{key}" + UUID.randomUUID();
+
+        client.set(key1, value1).get();
+        String data = client.getex(key1).get();
+        assertEquals(data, value1);
+
+        data = client.getex(key1, GetExOptions.Seconds(10L)).get();
+        Long ttlValue = client.ttl(key1).get();
+        assert ttlValue >= 0L;
```

Comment:
Don't use plain `assert`... it's not a unit test operation.  
```suggestion
        assertTrue(ttlValue >= 0L);
```